### DISCUSSION TITLE
Where do the dependency labels of the English spacy model come from ?

### DISCUSSION THREAD

**Discussion by VSegonne** [explosion/spaCy#7181] [Help: Other Questions] [answered]
Hi,
I'm using the `en_core_web_lg` spacy model for dependency parsing and I need to be sure where the set of dependency labels come from. 

The documentation stipulates that the model was trained on Ontonote. Yet, Ontonote is only annotated with syntactic constituents. 
Roaming around on multiple discussions : 
 - <https://github.com/explosion/spacy/issues/233>
 - <https://stackoverflow.com/questions/62399475/where-i-can-find-the-complete-list-of-spacy-dependency-parsing-labels-or-annotat>

It seems that ClearNLP was used to convert Ontonote to dependency parsed data, possibly following this paper : <https://www.mathcs.emory.edu/~choi/doc/cu-2012-choi.pdf>

This is not clear to me, can anyone confirm this ?
And if so, that would be nice to include it somewhere in Spacy's documentation.

Thanks !

**Comment by adrianeboyd** [comment, marked as answer]
This information is in the v2 docs here: https://v2.spacy.io/api/annotation. This page seems to have gone missing in the v3 docs and should be added back, with updates since a few of the details have changed in v3.

**Reply by VSegonne** [reply]
Alright thank you !
